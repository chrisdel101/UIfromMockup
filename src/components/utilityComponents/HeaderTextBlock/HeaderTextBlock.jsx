import React from 'react'
import styles from './index.css'

function BlockTextHeader(props) {
  return (
    <div className={`block-text-header`}>
      <div className="cursive-text">{props.cursive}</div>
      <div className="block-text">{props.block}</div>
    </div>
  )
}

export default BlockTextHeader
