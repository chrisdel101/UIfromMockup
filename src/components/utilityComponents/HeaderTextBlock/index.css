.cursive-text {
  font-family: cursive;
}
.block-text {
  text-transform: uppercase;
  font-weight: bolder;
}
.cursive-text {
  font-family: 'Kaushan Script', cursive;
}
.block-text {
  font-family: 'Montserrat', sans-serif;
}

.block-text-header {
  text-align: center;
  margin-top: 40px;
}
