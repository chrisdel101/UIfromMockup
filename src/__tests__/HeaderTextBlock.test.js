import React from 'react'
import HeaderTextBlock from '../components/utilityComponents/HeaderTextBlock/HeaderTextBlock.jsx'
import renderer from 'react-test-renderer'

test('create and match snapshot', () => {
  const component = renderer.create(<HeaderTextBlock variant="primary" />)

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
