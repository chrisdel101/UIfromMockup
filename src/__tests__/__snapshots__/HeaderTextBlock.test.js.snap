// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create and match snapshot 1`] = `
<div
  className="block-text-header"
>
  <div
    className="cursive-text"
  />
  <div
    className="block-text"
  />
</div>
`;
